src/ElasticSearch.php

```diff
@@ -183,14 +183,13 @@ protected function buildResultSet($elasticResult = []): ResultSet
         $total = 0;
         if ($elasticResult && isset($elasticResult['hits'])) {
             foreach ($elasticResult['hits']['hits'] as $document) {
-                $uris[] = $document['_id'];
+                $uris[$document['_id']] = $document['_source'];
```

Comment:
As mentioned here, this is dangerous: https://github.com/oat-sa/generis/pull/857#discussion_r533387806

Reply:
I did this in order to allow frontend to read uri from key value. I adjust this to FE requirements. I will check if I can do this differently.

Reply:
We cannot adjust data format from a third library to adapt it to the FE of the client application. The client application should normalize the data for FE. 

I recommend using objects

Reply:
I changed approach and resign from Uris in keys.